### DISCUSSION TITLE
bigfix/11.0/platform/Platform/Console/c_adjusting_bigfix_server_relays_2

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#157] [Q&A]
# bigfix/11.0/platform/Platform/Console/c_adjusting_bigfix_server_relays_2

To get the best performance from BigFix, you might need to adjust the server and the relays.

https://help.hcl-software.com/bigfix/11.0/platform/Platform/Console/c_adjusting_bigfix_server_relays_2.html

**Comment by lesgomba** [comment]
What happens when the Download Cache Size is reached? Is there a message issued to that effect in the relay log?
